README.md

```diff
@@ -11,7 +10,6 @@
 
 [![Slurp is GPL-3.0](https://img.shields.io/github/license/slurpcode/slurp.svg)](https://github.com/slurpcode/slurp/blob/main/LICENSE)
 [![This project is using Percy.io for visual regression testing](https://percy.io/static/images/percy-badge.svg)](https://percy.io)
-[![Codacy Badge](https://app.codacy.com/project/badge/Grade/4199d4805b2a473284988680cffe85db)](https://www.codacy.com/gh/slurpcode/slurp/dashboard)
```

Comment:
https://app.codacy.com/gh/slurpcode/slurp/dashboard

Need to get the correct working badge

Codacy runs with GitHub actions on PR

![Screenshot from 2024-10-14 01-43-02](https://github.com/user-attachments/assets/0fb0e68f-fd31-456d-a35f-006063deb9aa)

Reply:
ive added it back and it seems to be working pls review